src/app/shared/entry/info-tab-topic/edit-topic/edit-topic-dialog.component.html

```diff
@@ -57,22 +58,48 @@ <h1 mat-dialog-title>Select a Topic</h1>
           </div>
         </mat-radio-button>
       </mat-card>
-      <mat-card>
-        <mat-radio-button [value]="TopicSelectionEnum.AI" data-cy="topic-ai-selection-button">
-          <div>
-            <span class="bubble mr-2"><img class="site-icons-small" src="../../../assets/svg/ai-icon.svg" alt="AI generation icon" /></span>
-            <strong>AI</strong>
+      <mat-card *ngIf="entry.topicAI">
+        <mat-radio-button [value]="TopicSelectionEnum.AI" data-cy="topic-ai-selection-button" class="w-100">
+          <div fxLayoutAlign="space-between">
+            <div>
+              <span class="bubble mr-2"
+                ><img class="site-icons-small" src="../../../assets/svg/ai-icon.svg" alt="AI generation icon"
+              /></span>
+              <strong>AI</strong>
+            </div>
+            <span
+              *ngIf="entry.approvedAITopic"
+              class="bubble approved"
+              matTooltip="This AI topic was reviewed and selected by the {{ entryTypeMetadata.term }} owner"
+              ><mat-icon class="mat-icon-sm">checkmark</mat-icon> Approved</span
+            >
           </div>
           <div class="size-small gray-caption">Generated by AI using the content of your {{ entryTypeMetadata.term }}.</div>
-          <div [class]="entry.topicAI ? '' : 'gray-caption'">{{ entry.topicAI || 'Not Available' }}</div>
+          <!-- Display an info card if the AI topic was automatically set as the topic selection but it hasn't been approved by the user -->
+          <mat-card
+            *ngIf="!entry.approvedAITopic && entry.topicSelection === TopicSelectionEnum.AI"
+            class="alert-info mat-elevation-z size-small py-1 my-3"
+          >
+            <mat-icon class="mat-icon-sm">info</mat-icon> This topic has not been reviewed by the {{ entryTypeMetadata.term }} owner. Click
```

Comment:
First sentence says it hasn't been reviewed by the owner. But "you" are the (an) owner. I would say something like `You have not approved this topic. Click Save to confirm you have...`.